packages/flutter/lib/src/material/tabs.dart

```diff
@@ -993,18 +993,29 @@ class _TabBarState extends State<TabBar> {
     );
 
     if (widget.isScrollable) {
+      final ScrollBehavior configuration = ScrollConfiguration.of(context);
+      final ScrollPhysics physics = configuration.getScrollPhysics(context);
       _scrollController ??= _TabBarScrollController(this);
       tabBar = SingleChildScrollView(
         scrollDirection: Axis.horizontal,
         controller: _scrollController,
         child: tabBar,
+        physics: _NoImplicitScrollingPhysics(physics),
```

Comment:
Why are we disabling implicit scrolling for TabBars? It seems like if I have my focus on Tab A and I swipe to move focus to the next element I should land on Tab B even if it is currently offscreen, no?

Reply:
I seem to recall the correct behavior for tab bars being no-implicit scrolling. I will verify that this is the case

Reply:
I haven't been able to confirm this behavior, so I'll remove this work around for now and we can revist later